specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -1770,6 +1887,130 @@
                 "sku"
             ]
         },
+        "TopologyList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "readOnly": true,
+                    "items": {
+                        "$ref": "#/definitions/TopologyResource"
+                    }
+                },
+                "nextLink": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The URI to fetch the next page."
+                }
+            }
+        },
+        "TopologyResource": {
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "readOnly": true,
+                    "$ref": "#/definitions/TopologyResourceProperties"
+                }
+            },
+            "allOf": [{
+                    "$ref": "#/definitions/Resource"
+                },
+                {
+                    "$ref": "#/definitions/Location"
+                }
+            ]
+        },
+        "TopologyResourceProperties": {
+            "type": "object",
+            "properties": {
+                "calculatedDateTime": {
+                    "type": "string",
+                    "format": "date-time",
+                    "readOnly": true,
+                    "description": "The UTC time on which the topology was calculated"
+                },
+                "resources": {

```

Comment:
please rename this property to something else. It can potentially conflict with the Deep PUT common model that ARM team is working on enabling

Reply:
Changed to topologyResources